src/region_plane.cpp

```diff
@@ -174,6 +207,22 @@ void RegPlane::shape_update()
   if (ystyle == VARIABLE) yp = yscale * input->variable->compute_equal(yvar);
 
   if (zstyle == VARIABLE) zp = zscale * input->variable->compute_equal(zvar);
+
+  if (nxstyle == VARIABLE) normal[0] = xscale * input->variable->compute_equal(nxvar);
+
+  if (nystyle == VARIABLE) normal[1] = yscale * input->variable->compute_equal(nyvar);
+
+  if (nzstyle == VARIABLE) normal[2] = zscale * input->variable->compute_equal(nzvar);
+
+  // enforce unit normal
+
+  double rsq = normal[0] * normal[0] + normal[1] * normal[1] + normal[2] * normal[2];
```

Comment:
The common use case will be that none of the normal components are variables.  So it would be better if the conversion to a unit vector were done only once in that case
(as in the prior code).  Only do it here if one or more components are variables.